src/v3/src/transformer/i18n/transform.test.ts

```diff
@@ -42,6 +42,9 @@ jest.mock('./transformPasscodeHint', () => ({
 jest.mock('./transformWebAuthNSubmitButton', () => ({
   transformWebAuthNSubmitButton: () => () => ({}),
 }));
+jest.mock('./transformPasswordMatches', () => ({
+  transformPasswordMatches: () => () => ({}),
```

Comment:
You could add this function below to assert that it has been called, but its not a huge deal if it isn't added.

Reply:
Oh good call -- I will add it